apps/vth-frontend/src/app/[locale]/theme/[themeSlug]/page.tsx

```diff
@@ -122,9 +128,10 @@ const ThemePage = async ({ params: { locale, themeSlug } }: Params) => {
       <PageContent className="utrecht-custom-page-content">
         <Grid spacing="sm">
           <GridCell sm={12}>
-            <BreadcrumbWithBacklink
-              breadcrumbProps={{ navigationElements: breadcrumbNavigationElements }}
-              backlinkProps={{ title: parentElement?.title || 'Home', href: parentElement?.href || '/' }}
+            <Breadcrumbs
+              links={breadcrumbNavigationElements}
+              Link={Link}
+              backLink={{ label: parentElement?.label || 'Home', href: parentElement?.href || '/', current: false }}
```

Comment:
This new version has a back arrow (left chevron) when there is only one element. In the original requested design, that arrow should only be visible on the mobile version. Not on desktop.

<img width="255" alt="image" src="https://github.com/frameless/strapi/assets/24610692/11913561-d010-4a9f-8687-75c7db6771b6">

Reply:
Discussed on slack with @AliKdhim87: we'll check with UX before pushing these new styles for VTH